sorc/grid_tools.fd/regional_esg_grid.fd/pmat.f90

```diff
@@ -42,69 +44,72 @@ module pmat
 
 contains
 
-!=============================================================================
+!> Swap vectors
+!!
+!! @param d vector
+!! @param e vector
+!! @author R. J. Purser
 subroutine sswpvv(d,e)!                                                [swpvv]
-!=============================================================================
-! Swap vectors
-!-------------
 real(sp),    intent(inout) :: d(:), e(:)
 real(sp)                   :: tv(size(d))
-!=============================================================================
 tv = d; d = e; e = tv
 end subroutine sswpvv
-!=============================================================================
+
+!> dswpvv routine
+!! @author R. J. Purser
 subroutine dswpvv(d,e)!                                                [swpvv]
```

Comment:
Always list all params, in order.

Reply:
Which param is not in order?

Reply:
In other cases, in your doxygen changes.